dashboard/test/controllers/api/v1/projects/section_projects_controller_test.rb

```diff
@@ -46,16 +46,9 @@ class Api::V1::Projects::SectionProjectsControllerTest < ActionController::TestC
 
   test_user_gets_response_for(
     :index,
-    name: 'student can access their own section projects',
-    user: -> {@student},
-    params: -> {{section_id: @section.id}}
-  )
-
-  test_user_gets_response_for(
-    :index,
-    name: 'student cannot access another sections projects',
+    name: 'student cannot access section projects',
     response: :forbidden,
-    user: :student,
+    user: -> {@student},
```

Comment:
Still working on these tests?

Reply:
These tests are done. I'm just undoing a change I made in an earlier iteration of this: 
https://github.com/code-dot-org/code-dot-org/pull/31334 

I haven't started tests on the new parts yet (I'll make sure that is part of this PR)